examples/sdl3_renderer/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+cmake_minimum_required(VERSION 3.27)
+
+# Project setup
+project(clay_sdl3_renderer C)
+set(CMAKE_C_FLAGS_DEBUG "-g -Wall -Werror")
+set(CMAKE_C_FLAGS_RELEASE "-O3")
+
+include(FetchContent)
+set(FETCHCONTENT_QUIET FALSE)
+
+# Download SDL3
+FetchContent_Declare(
+	SDL
+	GIT_REPOSITORY https://github.com/libsdl-org/SDL.git
+	GIT_TAG preview-3.1.6
+	GIT_SHALLOW TRUE
+	GIT_PROGRESS TRUE
+)
+message(STATUS "Using SDL via FetchContent")
+FetchContent_MakeAvailable(SDL)
+set_property(DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/_deps/sdl-src" PROPERTY EXCLUDE_FROM_ALL TRUE)
```

Comment:
You probably want to use `<lowercaseName>_SOURCE_DIR` instead of `_debs/sdl-src` https://cmake.org/cmake/help/latest/module/FetchContent.html

Reply:
I did not know that this variable existed. Thanks. I'll need to revisit some of my projects now I think :smile: